modules/cluster-logging-release-notes-5.1.0.adoc

```diff
@@ -0,0 +1,63 @@
+[id="cluster-logging-release-notes-5-1-0"]
+= OpenShift Logging 5.1.0
+
+This release includes link:https://access.redhat.com/errata/RHSA-2021:2112[RHSA-2021:2112 OpenShift Logging Bug Fix Release 5.1.0].
+
+[id="openshift-logging-5-1-0-new-features-and-enhancements"]
+== New features and enhancements
+
+This release adds improvements related to the following components and concepts.
+
+* As a cluster administrator, you can use Kubernetes pod labels to gather log data from an application and send it to a specific log collector. You can gather log data by configuring the `inputs[].name.application.selector.matchLabels` element in the `ClusterLogForwarder` custom resource (CR) YAML file. You can also filter the gathered log data by namespace.
+(link:https://issues.redhat.com/browse/LOG-883[*LOG-883*])
+
+* This release adds the following new `ElasticsearchNodeDiskWatermarkReached` warnings to the OpenShift Elasticsearch Operator (EO):
+ - Elasticsearch Node Disk Low Watermark Reached
+ - Elasticsearch Node Disk High Watermark Reached
+ - Elasticsearch Node Disk Flood Watermark Reached
+
++
+The EO issues these warnings when it predicts that an Elasticsearch node will reach the `Disk Low Watermark`, `Disk High Watermark`, or `Disk Flood Stage Watermark` thresholds in the next 6 hours. This warning period gives you time to respond before the node reaches the disk watermark thresholds. The warning messages also provide links to the troubleshooting steps, which you can follow to help mitigate the issue. The EO applies the past several hours of disk space data to a linear model to generate these warnings.
+(link:https://issues.redhat.com/browse/LOG-1100[*LOG-1100*])
+
+* JSON logs can now be forwarded as JSON objects, rather than quoted strings, to either Red Hat's managed Elasticsearch cluster or any of the other supported third-party systems. Additionally, you can now query individual fields from a JSON log message inside Kibana which increases the discoverability of specific logs. For more information, see the link:https://docs.openshift.com/container-platform/4.7/logging/cluster-logging-exported-fields[`ClusterLogForwarder`] documentation.
+(link:https://issues.redhat.com/browse/LOG-785[*LOG-785*])
+
```

Comment:
@Srivaralakshmi Can you please add this entry to the New features and enhancements: 
* OpenShift Logging 5.1 now supports {product-title} 4.7 and later running on:
 - IBM Power Systems
 - IBM Z and LinuxONE
 
 @rolfedh Is it ok to just add the IBM content in Sri's PR or do we need a new one?

Reply:
/lgtm

Reply:
@Srivaralakshmi, would you add a release note for the above to your PR?
@SNiemann15  Please provide a link to the related engineering Jira task/story.

Reply:
Sure @rolfedh, I will add.

Reply:
@SNiemann15 Please answer these queries for me:

- Are there any known limitations or can P/Z users expect full support?
- Please provide a link to the related engineering Jira task/story

Thank you very much!

Reply:
@Srivaralakshmi @rolfedh I got confirmation from IBM Z QE that there are no limitations. 

@lsmcfadden Can you confirm for Power please.

Reply:
@SNiemann15  ( cc @pravin-dsilva ) there are no limitations for Power.

Reply:
Thanks @SNiemann15. I have added the text to the RN. PTAL.